app/models/manageiq/providers/lenovo/physical_infra_manager/refresher.rb

```diff
@@ -2,7 +2,7 @@ module ManageIQ::Providers::Lenovo
   class PhysicalInfraManager::Refresher < ManageIQ::Providers::BaseManager::Refresher
     def parse_legacy_inventory(ems)
       log_header = "MIQ_LENOVO(#{self.class.name}.#{__method__} Calling for [#{ems.name}])"
-      $log.info("#{log_header}")
+      $log.info(log_header)
```

Comment:
Prefer to just log the string directly, log_header isn't being used anywhere else in this method so no need for a var